src/Core/src/Handlers/SearchBar/SearchBarHandler.Windows.cs

```diff
@@ -50,12 +55,22 @@ public static void MapCharacterSpacing(SearchBarHandler handler, ISearchBar sear
 		[MissingMapper]
 		public static void MapIsTextPredictionEnabled(IViewHandler handler, ISearchBar searchBar) { }
 
-		[MissingMapper]
-		public static void MapMaxLength(IViewHandler handler, ISearchBar searchBar) { }
+		public static void MapMaxLength(SearchBarHandler handler, ISearchBar searchBar)
+		{
+			handler.NativeView?.UpdateMaxLength(searchBar, handler._queryTextBox);
+		}
 
 		[MissingMapper]
 		public static void MapIsReadOnly(IViewHandler handler, ISearchBar searchBar) { }
 
+		void OnLoaded(object sender, UI.Xaml.RoutedEventArgs e)
+		{
+			_queryTextBox = NativeView?.GetFirstDescendant<MauiTextBox>();
```

Comment:
How does this work? I don't see a custom AutoSuggestBox style anywhere. Do we need to port FormsAutoSuggestBoxStyle to Core?

Reply:
Is defined here https://github.com/dotnet/maui/blob/4d2c627908362a0aad282bf51e7105626911a01d/src/Core/src/Platform/Windows/Styles/MauiAutoSuggestBoxStyle.xaml